crates/net/eth-wire/Cargo.toml

```diff
@@ -64,6 +64,7 @@ proptest-derive = "0.3"
 default = ["serde"]
 serde = ["dep:serde", "smol_str/serde"]
 arbitrary = ["reth-primitives/arbitrary", "dep:arbitrary", "dep:proptest", "dep:proptest-derive"]
+optimism = ["reth-primitives/optimism"]
```

Comment:
Don't see the `"reth-primitives/optimism"` dependency being used here. We should verify that it isn't being used in `clabby/op-reth`.